package.json

```diff
@@ -75,7 +78,7 @@
     "semver": "^5.0.0",
     "style-loader": "^0.23.1",
     "svg-sprite-loader": "4.1.3",
-    "terriajs": "8.1.25",
+    "terriajs": "https://github.com/terriajs/terriajs#15751503bc6024b7e0cc0cab72f662fd9d32d3a7",
```

Comment:
To be changed once https://github.com/TerriaJS/terriajs/pull/6212 has been merged